admin/conference-template/params.data

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
Why this is not called params.json?

Reply:
I call it .data because it's not really a proper .json file. Json doesn't support comments, but I want there to be comments in the file so that it can be easily read and edited by a person.

This is also the reason I wrote a special parse_json function. the native json parser throws an error if there are comments in the file.

Reply:
Ok, so it is not a json file. What about to call it .properties. This is more standard and I guess python has a good way to load it. 

```
CONFERENCE_NAME=This is a the conference name
```